apps/web/src/components/(dashboard)/common/command-menu.tsx

```diff
@@ -53,6 +55,33 @@ export function CommandMenu({ open, onOpenChange }: CommandMenuProps) {
   const [search, setSearch] = useState('');
   const [pages, setPages] = useState<string[]>([]);
 
+  const [searchResults, setSearchResults] = useState<
+    { label: string; path: string; recipents: string[] }[]
+  >([]);
+
+  const debouncedSearchString = useDebouncedValue(search, 1000);
+  const { isLoading: isSearchingDocuments, refetch: findDocuments } =
+    trpc.document.searchDocuments.useQuery(
+      {
+        query: search,
+      },
+      {
+        enabled: search !== '',
+        onSuccess: (data) => {
```

Comment:
These hooks will be deprecated in future versions of react-query and so we should avoid them 😄